src/App.js

```diff
@@ -2,35 +2,27 @@ import React, { useEffect } from "react";
 import "./App.css";
 import { request } from "./backend-request";
 import Dashboard from "./components/Dashboard"
+import { AdminDashboard } from "./pages/AdminDashboard"
+import { store } from "./redux/store";
+import { Provider } from "react-redux";
 
 function App() {
   useEffect(() => {
     const fetchData = async () => { 
       const res = await request("applications")
       const data = await res.json();
-      console.log(data)
+      // console.log(data)
       return res;      
     };
     fetchData();
   });
   return (
-    <div className="App">
-      <Dashboard />
-      <header className="App-header">
-        <img className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Provider store={store}>
```

Comment:
as I mentioned above its better to move  below lines to index.js.. thank you
ReactDOM.render(
  <Provider store={store}>
    <App />
  </Provider>,
  document.getElementById("root")
);

Reply:
yes :)